src/components/admin/RightColAdmin.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <!-- colonne de droite -->
+        <div class="right-column column is-half">
+          <!-- <GlobalBal/> -->
```

Comment:
Not a real fan of commented blocks in committed code. You can keep them (I understand you want to keep them at hand, but are they needed in the committed code ?

Reply:
fixed in following commits